extensions/tw-colorsearch/package.json

```diff
@@ -21,16 +24,17 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.33.2"
+    "@raycast/api": "^1.79.0",
+    "culori": "^4.0.1"
   },
   "devDependencies": {
-    "@types/node": "~16.10.0",
-    "@types/react": "^17.0.28",
-    "@typescript-eslint/eslint-plugin": "^5.0.0",
-    "@typescript-eslint/parser": "^5.0.0",
+    "@raycast/eslint-config": "^1.0.11",
+    "@types/culori": "^2.1.1",
+    "@types/node": "^20.8.10",
+    "@types/react": "^18.3.3",
     "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
     "prettier": "^2.5.1",
+    "react": "^18.2.0",
```

Comment:
Since you've upgraded most dependencies, would you mind updating the `eslint` and `prettier` in the PR?

Reply:
I have updated the dependencies and the title. Thank you.